app/src/androidTest/java/android/andela/com/kenyajavadevs/view/MainActivityTest.java

```diff
@@ -25,15 +26,67 @@
     public ActivityTestRule<MainActivity> mMainActivityTestRule =
             new ActivityTestRule<>(MainActivity.class);
 
+//    private WifiManager wifiManager;
```

Comment:
Good job, consider removing these comments. There should be no comments on production.

Reply:
Thank you for the review, I've removed them.